ot/bregman.py

```diff
@@ -955,9 +966,23 @@ def get_reg(n):  # exponential decreasing
         log['alpha'] = alpha
         log['beta'] = beta
         log['warmstart'] = (log['alpha'], log['beta'])
-        return G, log
+        if output_value:
+            output = sinkhorn_stabilized(a, b, M, regi,
+                                         numItermax=numInnerItermax, stopThr=1e-9,
+                                         warmstart=(alpha, beta), verbose=False,
+                                         print_period=20, tau=tau, log=False)
+            return output, log
+        else:
+            return G, log
     else:
```

Comment:
is that an else for the loop? is that necessary? it is not covered by the tests  also.

Reply:
Still  +0.84%  coverage is awesome (also the bug was in this percent ;) )